rust/tools/authorization-logic/src/main.rs

```diff
@@ -0,0 +1,21 @@
+// try blocks are used in antlr4-rust
+#![feature(try_blocks)]
+
+mod ast;
+mod compilation_top_level;
+mod parsing;
+mod signing;
+mod souffle;
+mod test;
+
+use std::env;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
```

Comment:
This might benefit from using something like structuopt in future to support named command-line arguments and provide help etc. (https://docs.rs/structopt/0.3.21/structopt/)

Reply:
+1